### DISCUSSION TITLE
[Feature] Harmonyos support

### DISCUSSION THREAD

**Discussion by Mrzqd** [immich-app/immich#13571] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Will IMMICH-APP exit HarMonyOS's mobile application?
Harmonyos is Huawei's new operating system, and it is also Harmony-NEXT

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
APKs are attached to the github releases (https://github.com/immich-app/immich/releases).

**Reply by LstarXYX** [reply]
if there are any plan for Harmony Next OS app?
Harmony Next will only support .hap file and not support APKs
Thank you

**Reply by mss-x** [reply]
In 25 years, a large number of Asian and European users will migrate from Android to Harmonyos and OpenHarmony, and we need new system apps

**Reply by PrestarLin** [reply]
> APKs are attached to the github releases (https://github.com/immich-app/immich/releases).

it is not Android，it base on openharmony.It will support in.the future？欢迎中国的开发者们移植合并

**Comment by xu75** [comment]
Hoping too.

**Reply by duckeaty** [reply]
hoping

**Reply by gaospecial** [reply]
Hoping too.

**Reply by sushilei** [reply]
Hoping too

**Reply by WeslyYoman** [reply]
i hope it will support harmony os next too

**Reply by algaeReed** [reply]
me too.

**Reply by spaceychen** [reply]
me too

**Comment by ettingshausen** [comment]
Developers outside of China may find it difficult to purchase a HarmonyOS phone

**Comment by ettingshausen** [comment]
There is already a fork: https://github.com/dgh1818/immich_ohos

**Reply by xu75** [reply]
Cool, Thanks for sharing.

**Reply by spaceychen** [reply]
It looks very poorly adapted...
